packages/transport-memory/src/index.js

```diff
@@ -5,7 +5,7 @@ import Promise from 'bluebird';
 
 // standardise object to have json data spec
 function standardiseObject(obj) {
-  return JSON.parse(JSON.stringify(obj));
+  return obj;
```

Comment:
we did this to emulate a network-based serialization. things like arrays, dates etc will behave differently